src/redux/reducers/mainOptions.ts

```diff
@@ -265,11 +267,11 @@ export const mainOptionsReducer = (
       return action.payload === Language.Ru
         ? {
             ...state,
-            mainOptions: getLocalItem<IChart>(LocalStorageKey.MainOptions) || mainInitialValueRu,
+            mainOptions: getLocalItem<IChart>(LocalStorageKey.MainOptions) || getMainInitialValue(Language.Ru),
           }
         : {
             ...state,
-            mainOptions: getLocalItem<IChart>(LocalStorageKey.MainOptions) || mainInitialValueEn,
+            mainOptions: getLocalItem<IChart>(LocalStorageKey.MainOptions) || getMainInitialValue(Language.En),
           };
     }
```

Comment:
Probably we can simplify here as well:
mainOptions: getLocalItem<IChart>(LocalStorageKey.MainOptions) || getMainInitialValue(action.payload),

Reply:
I could have guessed, thanks :)
Done!